### ISSUE TITLE
[Roadmap] Kanban Pages

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#4818]
For various orders (builds / purchase / quotes / etc) it would be helpful to have a generic "kanban" view

- Can display this view as an option instead of existing "calendar" or "table" views
- Groups orders (etc) into pending / in progress / complete / cancelled

**Comment by matmair** [comment]
@SchrodingersGat is this still relevant?

**Comment by SchrodingersGat** [comment]
Yes I'm still interested in developing this - but I'll push to the 2.0.0 milestone
